includes/class-db.php

```diff
@@ -94,20 +94,23 @@ public static function update_options_key( $old, $new ) {
 	public static function migrate_to_1_0_0() {
 		// 1. rename comment meta
 		self::update_commentmeta_key( 'semantic_linkbacks_avatar', 'avatar' );
-		self::update_commentmeta_key( 'semantic_linkbacks_author_url', 'webmention_author_url' );
```

Comment:
This should migrate to $comment->comment_author_url

Reply:
I had a look at the code and this is mostly already done. Should I do it anyways?

Reply:
@dshanske `comment_author_url` currently has the brid.gy and `semantic_linkbacks_author_url` is the real URL. I think we have to re-check the code here to get that right!

Reply:
> @dshanske `comment_author_url` currently has the brid.gy and `semantic_linkbacks_author_url` the real URL. I think we have to re-check the code here to get that right!

https://github.com/pfefferle/wordpress-webmention/blob/5.X/includes/Entity/class-item.php#L388 - You can see right here 

https://github.com/pfefferle/wordpress-webmention/blob/5.X/includes/Entity/class-item.php#L383 - And here

So, the author_url will only be the Bridgy URL if it can't find anything else, as a fallback.

Reply:
And where do we store the bridgy URL?

Reply:
That is the source_url

Reply:
It is still registered as webmention_source_url